contrib/response.go

```diff
@@ -0,0 +1,104 @@
+// ⚡️ Fiber is an Express inspired web framework written in Go with ☕️
+// 🤖 Github Repository: https://github.com/gofiber/fiber
+// 📌 API Documentation: https://docs.gofiber.io
+
+package contrib
+
+import (
+	"fmt"
+	"net/http"
+	"unsafe"
+
+	"github.com/gofiber/fiber/base"
+)
+
+// Quickly to string
+func ToString(b []byte) string {
+	return *(*string)(unsafe.Pointer(&b))
```

Comment:
please use the method which is set by the app for string conversion, depending on whether the user has set immutable to true or false in the app, a different method is used internally 

but I don't think we need this methodology here